### PR TITLE
Leaf/pattycakesxd/f6.1 profile page picture and bio

### PR DESCRIPTION
**Pull request by PattyCakesxD**
Re-merges revert (#224)

Original PR (#221)

### CODE REVIEW

**Review by Leaversa — APPROVED** [review]

File: meshapp/src/profile/profile-page.tsx [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+const theme = createTheme({
+  palette: {
+    mode: "light",
+    primary: {
+      main: "#0b7d66",
+    },
+  },
+  typography: {
+    h1: {
+      fontFamily: "cocogoose",
```
Comment (GuyWhoCode) [inline meshapp/src/profile/profile-page.tsx:24]: Would it be possible to group the h1-h4 styles into a single common style using a CSS class name?

**Review by GuyWhoCode — APPROVED** [review]
Looks good with one minor styling adjustment.

**Comment by misslame** [comment]
Please retarget to `dev_new`

File: meshapp/src/profile/profile-page.tsx (hunk 1 above)
Comment (PattyCakesxD) [reply meshapp/src/profile/profile-page.tsx:24]: yep, done 👍

**Pull request merged**